src/index.js

```diff
@@ -0,0 +1,121 @@
+const state = {
+  Temperature: 0,
+  city: 'Seattle',
+  sky: 'sunny',
+  latitude: 0,
+  longitude: 0,
+};
+
+const increaseTemp = (event) => {
+  const tempContainer = document.querySelector('#Temperature');
+  console.log('increaseTemp clicked');
+  state.Temperature = Math.round(state.Temperature + 1);
+  tempContainer.textContent = `${state.Temperature}`;
+  tempRange();
+};
+// hello this is a new branch!! 
+
+const decreaseTemp = (event) => {
+  const tempContainer = document.querySelector('#Temperature');
+  state.Temperature = Math.round(state.Temperature - 1);
+  tempContainer.textContent = `${state.Temperature}`;
+  tempRange();
+};
+
+const tempRange = () => {
+  const tempContainer = document.querySelector('#Temperature');
+  const landscapeContainer = document.querySelector('#landscape');
+  if (state.Temperature >= 80) {
+    tempContainer.style.color = 'red';
+    landscapeContainer.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  } else if (state.Temperature <= 79 && state.Temperature >= 70) {
+    tempContainer.style.color = 'orange';
+    landscapeContainer.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else if (state.Temperature <= 69 && state.Temperature >= 60) {
+    tempContainer.style.color = 'yellow';
+    landscapeContainer.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (state.Temperature <= 59 && state.Temperature >= 50) {
+    tempContainer.style.color = 'green';
+    landscapeContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (state.Temperature <= 49) {
+    tempContainer.style.color = 'teal';
+    landscapeContainer.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  }
+};
```

Comment:
How might you refactor this so the logic is more data driven? Here you have several if/else if statements. What if you had an object with the keys as the temperature and the values as the `landscape` and `temperature` corresponding display? Then you could iterate over the object to set state!

It could look something like this:
```js
 const tempColors = [
   [49, 'blue'],
   [59, 'green'],
   [69, 'gold'],
   [79, 'orange'],
   [null, 'red'],
 ];
 
 const getColorForTemp = (temp) => {
   for (const [boundaryTemp, color] of tempColors) {
     if (boundaryTemp === null || temp <= boundaryTemp) {
       return color;
     }
   }
 };
  ```